pkg/discovery/helper.go

```diff
@@ -109,7 +109,7 @@ func (h *helper) Refresh() error {
 
 	h.resources = discovery.FilteredBy(
 		discovery.ResourcePredicateFunc(func(groupVersion string, r *metav1.APIResource) bool {
```

Comment:
@nrb if you wanted to, you could extract this into a separate function, and then unit test it by providing different `APIResource`s and ensuring that it returns true if and only if the resource supports all 4 verbs.

Reply:
That would make me a bit more confident in the change. I'll look into this.